app/assets/stylesheets/components/_profile_pages.scss

```diff
@@ -0,0 +1,72 @@
+.card-product {
```

Comment:
We already had a cards partial, with some things different than what le wagon's ui kitt has, for example, a much softer box-shadow and slightly rounded corners... I recommed you checking it out, so that we can keep on using the same components throught the different pages!

Reply:
ah ok I will unify my cards with the style of the other pages